b_pull_Landsat_SRST_poi/py/gee_functions.py

```diff
@@ -0,0 +1,674 @@
+def csv_to_eeFeat(df, proj, wrs):
+  """Function to create an eeFeature from the location info
+
+  Args:
+      df: point locations .csv file with Latitude and Longitude
+      proj: CRS projection of the points
+
+  Returns:
+      ee.FeatureCollection of the points 
+  """
+  features=[]
+  for i in (df.index):
+    x,y = df.Longitude[i],df.Latitude[i]
+    latlong =[x,y]
+    loc_properties = {"system:index": str(df.id[i]), 
+    "id": str(df.id[i]),
+    "wrs": str(wrs)}
+    g=ee.Geometry.Point(latlong, proj) 
+    feature = ee.Feature(g, loc_properties)
+    features.append(feature)
+  ee_object = ee.FeatureCollection(features)
+  return ee_object
+
+
+def apply_scale_factors(image):
+  """ Applies scaling factors for Landsat Collection 2 surface reflectance 
+  and surface temperature products
+
+  Args:
+      image: one ee.Image of an ee.ImageCollection
+
+  Returns:
+      ee.Image with band values overwritten by scaling factors
+  """
+  opticalBands = image.select("SR_B.").multiply(0.0000275).add(-0.2)
+  thermalBands = image.select("ST_B.*").multiply(0.00341802).add(149.0)
+  return image.addBands(opticalBands, None, True).addBands(thermalBands, None,True)
+
+
+def dp_buff(image):
+  """ Buffer ee.FeatureCollection sites from csv_to_eeFeat by user-specified radius
+
+  Args:
+      image: ee.Image of an ee.ImageCollection
```

Comment:
should the image variable be named feature/geometry to avoid confusion? also do we want a separate variable for inputting radius of the buffer?